<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EZData</string>
  </property>
  <property name="toolTip">
   <string notr="true" extracomment="click to choose, double click to cancel"/>
  </property>
  <property name="statusTip">
   <string notr="true" extracomment="click to choose, double click to cancel"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(230, 230, 230);
}

/* Style sheet for the buttons */
QPushButton {
	color: rgb(76, 76, 76);
	background-color: rgb(235, 235, 235);
	border: 1px solid rgb(153, 153, 153);
	border-radius: 4px;
	min-width: 24px;
	min-height: 24px;
	padding: 2px;
}

QPushButton:pressed {
    background-color: rgb(204, 204, 204);
}

QPushButton:default {
	border-color: rgb(76, 76, 76);
}

/* Style sheet for the data name list */
QListView {
    show-decoration-selected: 1;
 /* make the selection span the entire width of the view */
	margin: 8px;
	border-radius: 8px;
}

QListView::item:selected {
    border: 1px solid rgb(204, 204, 204);
}

QListView::item:selected:!active {
    background: rgb(230, 230, 230);
}

QListView::item:selected:active {
    background: rgb(204, 204, 204);
}

QListView::item:hover {
    background: rgb(230, 230, 230);
}

/* Style sheet for the QDockWidgets*/

QDockWidget { 
    titlebar-normal-icon: url(:icon/exit.svg);
}

QDockWidget::float-button  {
	top: 3px;
	right: 4px;
}

QDockWidget::title {
	padding: 1px;
	text-align: left;
	color: rgb(76, 76, 76);
	background-color: rgb(230, 230, 230);
	border: 1px solid rgb(153, 153, 153);
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_plot">
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="PlotWidget" name="plot_canvas">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame_plot_control">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	width: 24px;
	height: 24px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="1" column="9">
           <widget class="QPushButton" name="button_set_plot">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/icon/setting.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="button_edit_data">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/icon/edit.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="11">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="button_delete">
            <property name="styleSheet">
             <string notr="true">image: url(:/icon/cross.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="button_add_legend">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/icon/legend.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="button_resize">
            <property name="styleSheet">
             <string notr="true">image: url(:/icon/resize.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="button_import">
            <property name="styleSheet">
             <string notr="true">image: url(:/icon/import.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>button_edit_data</zorder>
         <zorder>horizontalSpacer</zorder>
         <zorder>button_add_legend</zorder>
         <zorder>button_set_plot</zorder>
         <zorder>button_resize</zorder>
         <zorder>button_delete</zorder>
         <zorder>button_import</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <property name="icon">
     <iconset theme="file">
      <normaloff>../../../../../../../.designer/backup</normaloff>../../../../../../../.designer/backup</iconset>
    </property>
    <widget class="QMenu" name="menu_import_file">
     <property name="statusTip">
      <string>Import the data using presets</string>
     </property>
     <property name="title">
      <string>Import Method</string>
     </property>
    </widget>
    <addaction name="menu_import_file"/>
    <addaction name="action_export_data"/>
    <addaction name="separator"/>
    <addaction name="action_import_setting"/>
   </widget>
   <widget class="QMenu" name="menu_data">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="action_edit_data"/>
    <addaction name="action_remove_data"/>
    <addaction name="action_clear_data"/>
   </widget>
   <widget class="QMenu" name="menu_addon">
    <property name="title">
     <string>Addon</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_data"/>
   <addaction name="menu_addon"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="windowTitle">
    <string>Data Name</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="file_list_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="click to choose, double click to cancel"/>
       </property>
       <property name="statusTip">
        <string extracomment="click to choose, double click to cancel"/>
       </property>
       <property name="whatsThis">
        <string extracomment="click to choose, double click to cancel"/>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_export_data">
   <property name="text">
    <string>Export data</string>
   </property>
  </action>
  <action name="action_export_figure">
   <property name="text">
    <string>Export figure</string>
   </property>
  </action>
  <action name="action_remove_data">
   <property name="text">
    <string>Remove Data</string>
   </property>
  </action>
  <action name="action_clear_data">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="action_import_setting">
   <property name="text">
    <string>Import Setting</string>
   </property>
  </action>
  <action name="actionElectrical_Chemistry_Tool">
   <property name="text">
    <string>Electrical Chemistry Tool</string>
   </property>
  </action>
  <action name="actionX_ray_Diffraction_Tool">
   <property name="text">
    <string>X-ray Diffraction Tool</string>
   </property>
  </action>
  <action name="actionPeak_fit_tool">
   <property name="text">
    <string>Fit tool</string>
   </property>
  </action>
  <action name="actionEnzyme_dynamic_Tool">
   <property name="text">
    <string>Enzyme Dynamic Tool</string>
   </property>
  </action>
  <action name="action_edit_data">
   <property name="text">
    <string>Edit Data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ezdata.qrc"/>
 </resources>
 <connections/>
</ui>
